ql/src/java/org/apache/hadoop/hive/ql/ddl/table/metaref/AlterTableCreateMetaRefAnalyzer.java

```diff
@@ -57,90 +59,55 @@ protected void analyzeCommand(TableName tableName, Map<String, String> partition
     DDLUtils.validateTableIsIceberg(table);
     inputs.add(new ReadEntity(table));
     validateAlterTableType(table, alterTableType, false);
-    AlterTableTypeReq alterTableTypeReq = new AlterTableTypeReq();
 
     String metaRefName = command.getChild(0).getText();
-    alterTableTypeReq.tableName = tableName;
-    alterTableTypeReq.metaRefName = metaRefName;
     Long snapshotId = null;
     Long asOfTime = null;
     Long maxRefAgeMs = null;
     Integer minSnapshotsToKeep = null;
     Long maxSnapshotAgeMs = null;
     AlterTableType alterTableType = command.getType()
```

Comment:
why do you need this? alterTableType is set in implementing classes

Reply:
Done.
Removed `AlterMetaRefOperationType` and Use `alterTableType ` which was set in implementation (branch & tag).
Thanks.